src/core/abstracts/SablierLockup.sol

```diff
@@ -51,10 +51,8 @@ abstract contract SablierLockup is
     /// @dev Emits a {TransferAdmin} event.
     /// @param initialAdmin The address of the initial contract admin.
```

Comment:
i am wondering if we still need this natspec comments if we already have it in `Adminable` contract?

Reply:
What do you mean? The `param` natspec is independent of that.

Reply:
the `param`natspec i agree, we should keep, but the `dev` one?

Reply:
Agree with you on that.

Reply:
You commented over `param` line so I thought thats you meant. Resolved in https://github.com/sablier-labs/v2-core/pull/1038/commits/5b24e38af6dbbd3ce876bd1b776fb9e7bdd06cf4.